ovos_backend_client/backends/ovos.py

```diff
@@ -0,0 +1,222 @@
+                   "units": units,
+                   "lang": lang}
+        url = f'{self.url}/weather/generate_current_weather_report/{self.uuid}'
+        response = self.post(url, data=reqdata, headers={"backend": "OWM"})
+        if response.status_code != 200:
+            raise RuntimeError(f"OWM api failed, status_code {response.status_code}")
+        response = response.json()
+        if "error" in response:
+            raise RuntimeError(response["error"])
+        return response
+
+    def owm_get_hourly(self, lat_lon=None, lang="en-us", units="metric"):
+        """Issue an API call and map the return value into a weather report
+
+        Args:
+            units (str): metric or imperial measurement units
+            lat_lon (tuple): the geologic (latitude, longitude) of the weather location
+        """
+        # default to configured location
+
+        lat, lon = lat_lon or self._get_lat_lon()
+        reqdata = {"lat": lat,
+                   "lon": lon,
+                   "units": units,
+                   "lang": lang}
+        url = f'{self.url}/weather/generate_hourly_weather_report/{self.uuid}'
+        response = self.post(url, data=reqdata, headers={"backend": "OWM"})
+        if response.status_code != 200:
+            raise RuntimeError(f"OWM api failed, status_code {response.status_code}")
+        response = response.json()
+        if "error" in response:
+            raise RuntimeError(response["error"])
+        return response
+
+    def owm_get_daily(self, lat_lon=None, lang="en-us", units="metric"):
+        """Issue an API call and map the return value into a weather report
+
+        Args:
+            units (str): metric or imperial measurement units
+            lat_lon (tuple): the geologic (latitude, longitude) of the weather location
+        """
+        raise KeyError("daily forecast API is non free")
+
+    # Wolfram Alpha API
+    def wolfram_spoken(self, query, units="metric", lat_lon=None, optional_params=None):
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        params = {'input': query,
+                  "geolocation": "{},{}".format(*lat_lon),
+                  'units': units,
+                  **optional_params}
+        url = f'{self.url}/wolframalpha/spoken/{self.uuid}'
+        data = self.post(url=url, params=params)
+        return data.text
+
+    def wolfram_simple(self, query, units="metric", lat_lon=None, optional_params=None):
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        params = {'input': query,
+                  "geolocation": "{},{}".format(*lat_lon),
+                  'units': units,
+                  **optional_params}
+        url = f'{self.url}/wolframalpha/simple/{self.uuid}'
+        data = self.get(url=url, params=params)
+        # TODO - this is returning an image ???
```

Comment:
@AIIX looks like a bug, this returned an image during testing and should be text